backend/main.py

```diff
@@ -10,61 +10,87 @@
 @recipeAPI.route('/api/addRecipe', methods=['POST'])
 def create_recipe():
     data = request.get_json()
+    id = data['id']
```

Comment:
these "id"'s will be called "usr_id" now, so this needs to change throughout the file